packages/commonwealth/server/controllers/server_communities_methods/create_community.ts

```diff
@@ -151,7 +151,10 @@ export async function __createCommunity(
   }
 
   // cosmos_chain_id is the canonical identifier for a cosmos chain.
-  if (community.base === ChainBase.CosmosSDK) {
+  if (
+    community.base === ChainBase.CosmosSDK &&
+    community.type !== ChainType.Offchain
+  ) {
```

Comment:
There is a block below with this conditional. You could move this section into that block instead.